### ISSUE TITLE
Add more info in amministrazioni.yml

### ISSUE THREAD

**Issue by sebbalex** [italia/developers-italia-backend#91]
According to [Generate pages for individual public entities](https://github.com/italia/developers.italia.it/issues/334) on [Developers Italia](https://developers.italia.it/) we need more info on `amministrazioni.yml` file which is currently produced by [this](https://github.com/italia/developers-italia-backend/blob/master/crawler/jekyll/amministrazioni.go#L58).

I suggest to add new fields in different steps:
```yaml
- ipa: PCM
  title: Presidenza del consiglio dei ministri
  publicAuthority:
    name: Presidenza del Consiglio di ministri
    styledName: <span class="font-weight-light">Presidenza del </span> Consiglio
```

later, with more info provided by onboarding:
```yaml
- ipa: unical
  title: Università della Calabria
  publicAuthority:
    name: UNIVERSITA' DELLA CALABRIA
    emblem: http://www.comune.bagnacavallo.ra.it/var/comune_bagnacavallo/storage/images/banner/banner-header-footer/stemma-header/1880-7-ita-IT/Stemma-Header.gif
    website: http://www.comune.bagnacavallo.ra.it
```

Note:
`styledName` is optional of course

**Comment by alranel** [comment]
Why do we need `publicAuthority`? Also, why do we need both `title` and `publicAuthority.name`? There could be just a single `name` item.

As of now, we don't have names or logos, so I think the current amministrazioni.yml should be enough (`ipa` and `name`), right?

**Comment by sebbalex** [comment]
`publicAuthority`was already used in jekyll template for PA's pages, it's a sort of legacy (we do not strictly need it). `name` and `title` were meant for page title and name of PA respectively, and yes, I agree, we do not need both. Other fields will be mandatory in next milestones.
We can close this issue for the moment.

**Comment by sebbalex** [comment]
Contrary to what I said before, we need a parent key to group all PA's attributes, this is due to how to Jekyll access to those data. Jekyll uses `page.ATTR` to access to them and some keys used for others purpose such as `title`, `name`, respectively used for: page title and page name (`page.html`).

Then we need a parent key to group attributes, we already have similar cases in `softwares.yml` where most of the attrs are under `pubbliccode` parent key, both to relate keys and to avoid overlapping as well. 

I expect that a correct file would be:

```yaml
- ipa: PCM
  title: Presidenza del consiglio dei ministri
  publicAuthority:
    name: Presidenza del Consiglio di ministri
```
where `title` will be used for page title and some browser utility (bookmarks, tab name, etc..) and `name` in the content header of PA's page.

**Comment by alranel** [comment]
Also, make sure that iPA codes are always lowercase without duplicates.

**Comment by alranel** [comment]
As discussed, I'm going for this structure:

```yaml
- ipa: pcm
  entityName: Presidenza del Consiglio dei Ministri
```

This guarantees that no clash happens.

I also created this PR that allows to use `entityName` as page title: https://github.com/italia/developers.italia.it/pull/381

**Comment by alranel** [comment]
Done in 9a4a7589b0c6b960447f1f80134bf4f098bdbfee and 3ef88b4ccd6c580d1768be306b5c4edaa3cbc5e4.

**Comment by sebbalex** [comment]
Fixed by #390
